AEPLifecycle/Sources/LifecycleV2/LifecycleV2Constants.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Constants for `LifecycleV2`
+enum LifecycleV2Constants {
+
+    static let STATE_UPDATE_TIMEOUT_SEC = TimeInterval(2)
+    static let CACHE_TIMEOUT_SECONDS = TimeInterval(2)
+
+    enum EventType {
+        static let APP_LAUNCH = "application.launch"
+        static let APP_CLOSE = "application.close"
+    }
```

Comment:
These two values need to be moved to Core EventType

Reply:
These values are set as part of XDM data and don't refer to event.type.

Reply:
Ahh... confused them with the actual event types, can we rename the enum to XDMEventType then?

Reply:
I prefer moving it inside LifecycleMetricsBuilder as we only use it inside that class.